api/src/main/java/io/opentelemetry/trace/EndSpanOptions.java

```diff
@@ -48,17 +46,18 @@ static EndSpanOptions getDefault() {
    * @since 0.1
    */
   public static Builder builder() {
-    return new AutoValue_EndSpanOptions.Builder();
+    return new AutoValue_EndSpanOptions.Builder().setEndTimestamp(0);
```

Comment:
Wouldn't this default to `0` anyway?

Reply:
Autovalue requires to set all the primitive values, this is essentially to make autovalue happy.